examples/vpc-scenario-1/main.tf

```diff
@@ -41,15 +41,18 @@ module "vpc" {
   region      = "${var.region}"
   cidr        = "10.23.0.0/16"
   azs         = ["${slice(data.aws_availability_zones.available.names, 0, 3)}"]
-  extra_tags  = { kali = "ma" }
-  public_subnet_cidrs  = ["${var.public_subnet_cidrs}"]
 
+  extra_tags = {
+    kali = "ma"
+  }
+
+  public_subnet_cidrs = ["${var.public_subnet_cidrs}"]
```

Comment:
I prefer the previous, which I see as the shorter, more concise and aligned version.